### PR TITLE
add copyright property in sre module

### PR DESCRIPTION
**Pull request by sm136599**
* add copyright property in sre module
-------------
result
```
>>>>> import _sre
>>>>> _sre.copyright
' SRE 2.2.2 Copyright (c) 1997-2002 by Secret Labs AB '
```

### CODE REVIEW

File: vm/src/stdlib/sre.rs [hunk 1]
```diff
@@ -28,6 +28,12 @@ mod _sre {
     #[pyattr]
     pub use sre_engine::{constants::SRE_MAGIC as MAGIC, CODESIZE, MAXGROUPS, MAXREPEAT};
 
+    #[pyattr]
+    fn copyright(_vm: &VirtualMachine) -> &str {
+        let copy_right = " SRE 2.2.2 Copyright (c) 1997-2002 by Secret Labs AB ";
+        copy_right
+    }
+
```
Comment (Snowapril) [inline vm/src/stdlib/sre.rs:36]:
```suggestion
    #[pyattr(name = "copyright")]
    const COPYRIGHT : &str = " SRE 2.2.2 Copyright (c) 1997-2002 by Secret Labs AB ";
```

Comment (Snowapril) [reply vm/src/stdlib/sre.rs:36]: Oh, I missed that... But It seems also true because most of the things offered by `sre.rs` and `_sre.c` are the same.

**Comment by youknowone** [comment]
@qingshi163 as i understand, this is mostly your work, right?

File: vm/src/stdlib/sre.rs (hunk 1 above)
Comment (youknowone) [reply vm/src/stdlib/sre.rs]: That means the API is the same. It doesn't necessarily mean the code is directly translation from one source to the other one. Though we now have unclear points due to google vs oracle case, basically copyright came from copy of code, not from copy of signatures.

**Comment by qingshi163** [comment]
> @qingshi163 as i understand, this is mostly your work, right?

Yes, the implements is heavily inspired by cpython as we use the same frontend, but I don't think there is direct translation.

**Comment by youknowone** [comment]
thank you for checking

**Pull request closed without merging**